.ci/checker-framework-suppressions/checker-formatter-suppressions.xml

```diff
@@ -1,28 +1,28 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressedErrors>
   <checkerFrameworkError unstable="false">
-    <fileName>checkstyle/src/main/java/com/puppycrawl/tools/checkstyle/LocalizedMessage.java</fileName>
```

Comment:
update of filename is required, since we do not include `checkstyle` directory any longer.